### ISSUE TITLE
Receiving >20 bytes at once from Bluefruit Connect app not working

### ISSUE THREAD

**Issue by dhalbert** [adafruit/Adafruit_CircuitPython_BLE#94]
@ Thornhill in discord says a UART message of  >20 bytes from the Bluefruit Connect app to, say, `ble_uart_echo_test`, is dropped. Sending to/from two CircuitPython boards does work. Also, does work sending to Arduino.

Confirmed by @caternuson.

**Comment by caternuson** [comment]
Yep. Just adding results of my simple and brief test. Phone is Android. BLE Connect v3.3.2. Running this example:
https://github.com/adafruit/Adafruit_CircuitPython_BLE/blob/master/examples/ble_uart_echo_test.py

![Screenshot_2020-06-30-10-25-14](https://user-images.githubusercontent.com/8755041/86158798-2aa17b80-babe-11ea-9202-2adf3707c85b.png)

```python
Adafruit CircuitPython 5.3.0 on 2020-04-29; Adafruit ItsyBitsy nRF52840 Express with nRF52840
>>> import ble_uart
b't'
b'e'
b's'
b't'
b'\n'
b't'
b'e'
b's'
b't'
b'2'
b'\n'
```

**Comment by tekktrik** [comment]
Aha!  I'm working on https://github.com/adafruit/Adafruit_CircuitPython_BluefruitConnect/issues/16 and this seems to be blocking that, given the size of images.  My testing confirms what @caternuson has described.

**Comment by tekktrik** [comment]
@dhalbert and @caternuson I can also send longer UART between boards.  Should this be transferred to the app's repo?

**Comment by dhalbert** [comment]
Maybe this is due to the app. I was afraid it is more general. There was this bug: https://github.com/adafruit/circuitpython/issues/3975, but it's now fixed.

**Comment by dhalbert** [comment]
This is due to the default `max_length` for variable-length Characteristics being `20`. It would be fixed by #37.

I looked at a couple of implementations of NUS (Nordic UART Service). The max length of the characteristics is not fixed by the NUS definition. Instead, it's chosen at build time, based on the maximum length on the MTU size - 3.
